src/app/forms/custom-sections/weights/weights.component.ts

```diff
@@ -103,20 +103,23 @@ export class WeightsComponent implements OnInit, OnDestroy, OnChanges {
   }
 
   private handleVehicleTechRecordChange(changes: SimpleChanges): void {
-    if (changes['vehicleTechRecord'] && !this.ladenWeightOverride && this.form) {
-      const { currentValue, previousValue } = changes['vehicleTechRecord'];
+    const { vehicleTechRecord } = changes;
+    if (this.form && vehicleTechRecord && !this.ladenWeightOverride) {
+      const { currentValue, previousValue } = vehicleTechRecord;
 
       const fieldsChanged = [
```

Comment:
You've missing the standing capacity from the fields change. With it removed, it isn't going to re-calculate when it changes